profile/2018-06-01-profile.json

```diff
@@ -0,0 +1,655 @@
+                         "2018-04-01": [
+                             "applicationGatewayAvailableSslOptions",
+                             "applicationGatewayAvailableWafRuleSets",
+                             "applicationGateways",
+                             "applicationGateways",
+                             "applicationSecurityGroups",
+                             "bgpServiceCommunities",
+                             "connections",
+                             "ddosProtectionPlans",
+                             "expressRouteCircuits",
+                             "expressRouteServiceProviders",
+                             "loadBalancers",
+                             "localNetworkGateways",
+                             "locations",
+                             "locations/CheckDnsNameAvailability",
+                             "locations/operationResults",
+                             "locations/operations",
+                             "locations/usages",
+                             "locations/virtualNetworkAvailableEndpointServices",
+                             "networkInterfaces",
+                             "networkSecurityGroups",
+                             "networkWatchers",
+                             "operations",
+                             "publicIPAddresses",
+                             "routeFilters",
+                             "routeTables",
+                             "virtualNetworkGateways",
+                             "virtualNetworks"
+                         ]
+        },
+
+        "microsoft.notificationHubs": {
+
+                         "2016-03-01": [
+                             "operationResults",
+                             "operations"
+                         ],
+                         "2017-04-01": [
+                             "checkNamespaceAvailability",
+                             "namespaces",
+                             "namespaces/notificationHubs"
+                         ]
+        },
+
+        "microsoft.policyInsights": {
+
+                         "2018-04-04": [
+                             "operations",
+                             "policyEvents",
+                             "policyStates"
+                         ]
+
+        },
+
+        "microsoft.powerBI": {
+
+                         "2016-01-29": [
+                             "locations",
+                             "locations/checkNameAvailability",
+                             "workspaceCollections"
+                         ]
+        },
+
+        "microsoft.recoveryServices": {
+
+                         "2016-06-01":[
+                             "locations",
+                             "locations/allocateStamp",
+                             "locations/allocatedStamp",
+                             "locations/backupStatus"
+
+                         ],
+                         
+                         "2016-10-08": [
+                             "operations"
```

Comment:
Based on ARM providers list, this does not exist, but there is:
['2016-08-10', '2016-06-01', '2015-12-15', '2015-12-10', '2015-11-10', '2015-08-15', '2015-08-10', '2015-06-10', '2015-03-15']